incubator/mattermost-team-edition/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for mattermost-team-edition.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: mattermost/mattermost-team-edition
+  tag: 5.0.0
+
+tags:
+  ingress: false
+
+# Mattermost configuration:
+config:
+  SiteUrl: "http://mattermost.example.com"
+  SiteName: "Mattermost"
+  FilesAccessKey:
+  FilesSecretKey:
+  FileBucketName:
+  SMTPHost:
```

Comment:
it seem this should be `SMTPServer` according to config.tpl

Reply:
done